src/current/v25.1/time.md

```diff
@@ -191,23 +191,15 @@ ALTER TABLE
 (2 rows)
 ~~~
 
-{{site.data.alerts.callout_info}}
-If a non-default precision level has already been specified, you cannot change the precision to a lower level.
-{{site.data.alerts.end}}
+You can reduce the precision of the values of `DECIMAL`, `TIMESTAMP`, and `TIMESTAMPTZ` columns; this yields the same behavior as the PostgreSQL implementation.
 
-In this case, the `time_val` column, which is of type `TIME(5)`, cannot be changed to a precision level below `5`:
+For example, to lower the precision of the `time_val` column, which is of type `TIME(5)`, use the following statement:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> ALTER TABLE time ALTER COLUMN time_val SET DATA TYPE TIME(3);
+ALTER TABLE time ALTER COLUMN time_val SET DATA TYPE TIME(3);
```

Comment:
Should this use the table `time_precise`?

Reply:
yes thanks! updated, PTAL